app/pubspec.yaml

```diff
@@ -52,7 +52,9 @@ dependencies:
   flutter_keyboard_visibility: ^5.4.0
   expandable_text: ^2.3.0
   cached_network_image: ^3.2.1
-  flutter_mentions: ^2.0.1
+  flutter_mentions:
+    git:
+      url: https://github.com/gtalha07/flutter_mentions.git
```

Comment:
what is happening here?

Reply:
Currently, ```flutter_mentions``` highly requested feature was to include ```suggestionListWidth``` so we can change the width of suggestion list container. I did that in my forked version as its needed for making it adaptive across different screen sizes. I am looking to open contribution PR for it but also seeing the repository being not actively maintained .i.e. last was 2 yrs ago...

Reply:
Seems like the [PR](https://github.com/fayeed/flutter_mentions/pull/105) is already opened for review and merge, but  no updates on it yet...

Reply:
thanks for explanation. If we need to maintain a fork, we probably want to move it into our org though ...